packages/legacy-workbench/src/js/angular/security/templates/modal/default-authorities.html

```diff
@@ -31,7 +31,7 @@ <h5 class="boldless col-xs-9">{{'security.repository.title' | translate}}</h5>
                 <span class="fa-kit fa-gdb-graphql fa-1andAhalfx col-xs-1" gdb-tooltip="{{'security.tooltip.graphql' | translate}}"
                       tooltip-placement="top" style="margin-top: 10px"></span>
             </div>
-            <div id="access-user-repos access-user-rights" class="row"
+            <div id="access-user-repos access-user-rights" class="row repository-rights"
```

Comment:
What the hell is this id?! Can you please check if it anything relies on it and fix it or remove it completely?